### ISSUE TITLE
HCL report run from shell yields "unary operator expected"

### ISSUE THREAD

**Issue by miczyg1** [Dasharo/dasharo-issues#736]
### Device

Any

### Dasharo version

v1.2.18 and newer (mayeb odler as well)

### Affected component(s) or functionality

Dasharo HCL report in DTS

### Brief summary

DEPLOY_REPORT variable is not exported when dasharo-hcl-report is executed from DTS shell

### How reproducible

Always

### How to reproduce

Run `dasharo-hcl-report ` from DTS shell

### Expected behavior

`unary operator expected` errors don't show up

### Actual behavior

`unary operator expected` error show up

### Screenshots

```
root@DasharoToolsSuite:~# dasharo-hcl-report 
/usr/sbin/dasharo-hcl-report: line 61: [: =: unary operator expected
/usr/sbin/dasharo-hcl-report: line 143: [: =: unary operator expected
CRITICAL ERROR: cannot dump firmware                               |
###################################################################|

/usr/sbin/dasharo-hcl-report: line 269: [: =: unary operator expected
/usr/sbin/dasharo-hcl-report: line 294: [: =: unary operator expected
/usr/sbin/dasharo-hcl-report: line 304: [: =: unary operator expected
```

**Comment by TomaszAIR** [comment]
@miczyg1 fix for that will be added to v1.2.20. Right now I have rc4 version on which it works. Tested on qemu:

```
# dasharo-hcl-report 
Getting hardware information. It will take a few minutes...
Sound card files are missing!                                      |
CRITICAL ERROR: cannot dump firmware                               |
###################################################################|

[OK]            PCI configuration space and topology
[UNKNOWN]       USB devices and topology
[OK]            Super I/O configuration
[UNKNOWN]       EC configuration
[ERROR]         MSRs
[OK]            SMBIOS tables
[OK]            BIOS information
[OK]            CMOS NVRAM
[UNKNOWN]       Intel configuration registers
[OK]            GPIO configuration C header files
[OK]            kernel dmesg
[OK]            ACPI tables
[UNKNOWN]       Audio devices configuration
[OK]            CPU info
[OK]            I/O ports
[OK]            Input bus types
[ERROR]         Firmware image
[OK]            I2C bus
[UNKNOWN]       ACPI tables
[OK]            Touchpad information
[ERROR]         DIMMs information
[ERROR]         CBMEM table information
[UNKNOWN]       TPM information
[ERROR]         AMT information
[OK]            ME information
Results of getting data:

Legend:
[OK]             Data get successfully
[UNKNOWN]        Result is unknown
[ERROR]          Error during getting data

Creating archive with logs...
Done! Logs saved to: /QEMU_Standard_PC_(Q35_+_ICH9,_2009)_0version_754df0b9-bf8e-5beb-bc03-95d22bfae38c_2024_03_13_13_28_26_076265609.tar.gz
```

Tested version available here http://boot.dasharo.com/dts/dts-rc.ipxe. USB bootable image can be downloaded from http://boot.dasharo.com/dts/v1.2.20-rc4/

**Comment by artur-rs** [comment]
Release v1.2.20 is now live - see the changelog [here](https://github.com/Dasharo/meta-dts/blob/main/CHANGELOG.md#v1220---2024-03-19). Closing.
